docs/identifierCase.md

```diff
@@ -0,0 +1,58 @@
+# identifierCase
+
+Converts identifiers to upper or lowercase.
+
+Note: An identifier is a name of a SQL object.
+There are two types of SQL identifiers: ordinary identifiers and quoted identifiers.
+Only ordinary identifiers are subject to be converted.
```

Comment:
This is better, but according to this description I would expect the case of variables and parameters to also be converted, because they too are names of SQL objects.

In general I see all these as different kinds of identifiers:

- schema, table and column names
- variable names
- function names
- parameter names

Variables are a particularly tricky case. Some dialects like MySQL have variables with a prefix like `@foo`, and therefore we can easily distinguish them. Other dialects like PostgreSQL also have variables, but there is no special prefix, an idientifier `foo` could refer to a variable, or it could refer to a table or column.

Reply:
Ah, I think I mixed variables and parameters... by variables, do you mean SQL variables defined by `CREATE VARIABLE` (in DB2)? If yes, then I agree that they should also be considered an identifier and converted.

Reply:
Yes, variables like that.